client/src/components/App6SymbolPreview.tsx

```diff
@@ -0,0 +1,126 @@
+import { Popover, PopoverInteractionKind } from "@blueprintjs/core"
+import App6Symbol, { getChoices } from "components/App6Symbol"
+import { Organization } from "models"
+import React from "react"
+import { Table } from "react-bootstrap"
+import Settings from "settings"
+
+interface FieldRow {
+  fieldName: string
+  values: any[]
+  parentValues: object
+}
+
+const FieldRow = ({ fieldName, values, parentValues }: FieldRow) => {
+  const mergedValues = { ...values }
+  Object.entries(parentValues).forEach(([key, value]) => {
+    if (value !== null && mergedValues[key] === null) {
+      mergedValues[key] = value
+    }
+  })
+  if (!mergedValues[fieldName]) {
+    return null
+  }
+  const choices = getChoices(fieldName, mergedValues)
+  const text =
+    choices[mergedValues[fieldName]] + (values[fieldName] ? "" : " (inherited)")
+  return (
+    <tr style={{ border: "hidden" }}>
+      <td style={{ fontWeight: "bold" }}>
+        {Settings.fields.organization[fieldName].label}
+      </td>
+      <td id={fieldName}>{text}</td>
+    </tr>
+  )
+}
+
+interface App6SymbolPreviewProps {
+  values: any
+  size?: number
+  maxHeight?: number
+}
+
+const App6SymbolPreview = ({
+  values,
+  size = 30,
+  maxHeight
+}: App6SymbolPreviewProps) => {
+  const { parentContext, parentStandardIdentity, parentSymbolSet } =
+    Organization.getApp6ParentFields(values, values)
+  const parentValues = {
+    app6context: parentContext,
+    app6standardIdentity: parentStandardIdentity,
+    app6symbolSet: parentSymbolSet
+  }
+  const app6ValueKeys = Object.keys(values).filter(key =>
+    key.startsWith("app6")
+  )
+
+  return (
+    <Popover
+      captureDismiss
+      interactionKind={PopoverInteractionKind.HOVER}
+      placement="bottom-start"
+      autoFocus
+      enforceFocus={false}
+      content={
+        <div
+          style={{
+            width: 500,
+            backgroundColor: "#e0e8ed",
+            padding: 20
+          }}
+        >
+          <h4 className="mb-3">APP-06 Symbology</h4>
+          <div
+            className="bg-white"
+            style={{
+              borderRadius: 4,
+              padding: 20
+            }}
+          >
+            <Table responsive>
+              <tbody>
+                {app6ValueKeys.map(key => {
+                  return (
+                    <FieldRow
+                      key={key}
+                      fieldName={key}
+                      values={values}
+                      parentValues={parentValues}
+                    />
+                  )
+                })}
```

Comment:
Change to:
```javascript
                {app6ValueKeys.map(key => (
                  <FieldRow
                    key={key}
                    fieldName={key}
                    values={values}
                    parentValues={parentValues}
                  />
                ))}
```
(same as https://github.com/NCI-Agency/anet/pull/5160#discussion_r2055897277 )